### PR TITLE
Added sizing information and kube-node.yml example

### PR DESCRIPTION
**Pull request by wabernat**
Adds storage configuration information, as well as sample kube-node.yml values.

**Which issue does this PR fix?**

fixes #ZENKO-1190

**Special notes for your reviewers**:

### CODE REVIEW

File: docs/Installation/Installation_Guide.rst [hunk 1]
```diff
@@ -102,6 +133,41 @@ If you are behind a proxy, add the following lines to your local machine’s
...
+    lv03:
+      size: 125G
+    lv04:
+      size: 62G
+    lv05:
+      size: 62G
+
+For custom sizing, increase these base numbers.
+
+For non-MetalK8s deployments, follow your vendo or community’s instructions for
```
Comment (ssalaues) [inline docs/Installation/Installation_Guide.rst]: `vendo` vs `vendor` otherwise LGTM

Comment (wabernat) [reply docs/Installation/Installation_Guide.rst]: fixed

**Review by ssalaues — APPROVED** [review]

**Comment by ssalaues** [comment]
@bert-e reset

**Comment by ssalaues** [comment]
@bert-e force_reset

**Comment by ssalaues** [comment]
@bert-e  reset

**Review by alexanderchan-scality — APPROVED** [review]

**Pull request merged**